modules/apps/site/site-cms-site-initializer/src/main/java/com/liferay/site/cms/site/initializer/internal/frontend/data/set/view/cards/FilesSectionCardsFDSView.java

```diff
@@ -26,6 +26,11 @@ public String getDescription() {
 		return StringPool.BLANK;
 	}
 
+	@Override
+	public String getImage() {
+		return "embedded.file.thumbnailSrc";
```

Comment:
@Mikellorza, are we embedding it? I don't see it in the DTO Converter

Reply:
yes, here : https://github.com/liferay-content-management/liferay-portal/pull/6581/commits/74c88cc9c2dd3be2cae92c1074b0beee2d96d124

Reply:
see: https://github.com/liferay/liferay-portal/blob/0c0f9dfc0bdca5fd6d6ec4da770209e6b9e99485/modules/apps/portal-search/portal-search-rest-impl/src/main/java/com/liferay/portal/search/rest/internal/resource/v1_0/SearchResultResourceImpl.java#L477